about.html

```diff
@@ -0,0 +1,153 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300;400;500;600;700&family=Roboto&display=swap"
+    rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Rubik:wght@500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap" rel="stylesheet">
+
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <title>About</title>
+</head>
+<body>
+  
+  <div class="sidebar">
+    <ul class="dsk-nav">
+      <li class="closeb"><i class="fas fa-times-circle"></i></li>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="">Program</a></li>
+      <li><a href="">Join</a></li>
+      <li><a href="">Sponsor</a></li>
+      <li><a href="">News</a></li>
+    </ul>
+  </div>
+  <header class="desktop-header">
+    <ul class="online-icons">
+      <li><a class="iconx fb" href="/">
+        <i class="fab fa-facebook-square"></i>
+        </a>
+      </li>
+      <li> <a class="iconx tw" href="/">
+        <i class="fab fa-twitter"></i>
+      </a></li>
+      <li class="iconx en"> <a class="iconx" href="">English</a></li>
+    </ul>
+  </header>
+
+  <div class="desktop-nav">
+    <img src="./images/firstnavlogo.png" alt="">
+    <ul class="dsk-nav-items">
+      <li class="nav1"><a class="nav1" target="_self" href="index.html">Home</a></li>
+      <li class="nav2">Program</li>
+      <li class="nav3" >Sponsor</li>
+      <li class="nav4" >News</li>
+    </ul>
+    <button class="last-logo">CC Campaign</button>
+  </div>
```

Comment:
- [ ] The **font family**, **font size**, ... of your _white navbar_ section does not match the provided template design. Kindly ensure to improve upon these aspects

**Your navbar:**
![image](https://user-images.githubusercontent.com/12745474/133619154-bcc8d011-2ae9-4ebc-b9ec-1d0a3199159c.png)

**The original navbar:**
![image](https://user-images.githubusercontent.com/12745474/133619326-24009082-be8f-42bb-8f3d-469d9748353f.png)

Reply:
This issue is not yet resolved!